packages-next/auth/src/types.ts

```diff
@@ -52,7 +52,7 @@ export type AuthConfig<GeneratedListTypes extends BaseGeneratedListTypes> = {
     /** Array of fields to collect, e.g ['name', 'email', 'password'] */
     fields: GeneratedListTypes['fields'][];
     /** Suppresses the second screen where we ask people to subscribe and follow Keystone */
-    skipKeystoneSignup?: boolean;
+    skipKeystoneWelcome?: boolean;
```

Comment:
I think this makes this into a `major` version bump, since it's potentially a breaking change for users.

Reply:
yep, good point 
Added a major changeset to account for this 👍